packages/default/scss/action-sheet/_theme.scss

```diff
@@ -0,0 +1,51 @@
+@include exports("action-sheet/theme") {
+
+    .k-action-sheet {
+        @include box-shadow( $action-sheet-box-shadow );
+
+
+        .k-listgroup-item {
+            background: $action-sheet-bg;
+
+            &.k-action-sheet-header {
+                opacity: .5;
+            }
+            + .k-listgroup-item {
+                border-color: transparent;
+            }
+        }
+
+        .k-link:hover,
+        .k-link.k-state-hover {
+            background: $action-sheet-item--hovered-background 0% 0% no-repeat padding-box;
```

Comment:
Please, do not use shorthands, check this - https://github.com/telerik/kendo-themes/wiki/Code-Styleguide#shorthand-notation

Shorthands are hard to read, support, they can override something that should not be overridden, etc.